### ISSUE TITLE
ALIAS record doesn't work with cloudflare_proxy

### ISSUE THREAD

**Issue by bfirsh** [StackExchange/dnscontrol#95]
```
D("fir.sh", REG_NONE, DnsProvider(CFLARE),
  ALIAS('@', 'fir.sh.s3-website-us-east-1.amazonaws.com.', {"cloudflare_proxy": "ON"}),
...
```

throws:

```
$ dnscontrol preview
...
******************** Domain: fir.sh
----- Getting nameservers from: cloudflare.com
----- DNS Provider: cloudflare.com... ERROR
Error getting corrections: cloudflare_proxy set on ALIAS record: "@" cloudflare_proxy="ON"
...
```

**Comment by captncraig** [comment]
Ah, a bit of validation I missed.

**Comment by captncraig** [comment]
Should be fixed in 4f0b6059081eee205f6519fac55b0d09440fdaa2

**Comment by bfirsh** [comment]
Sweet - works a treat. Thank you! :D
